api/source/service/mysql/MigrationHandler.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
```

Comment:
Please place this file at `api/source/service/mysql/migrations/lib/MigrationHandler.js`